packages/@aws-cdk/custom-resources/test/provider-framework/integration-test-fixtures/s3-file-handler/index.ts

```diff
@@ -56,7 +56,7 @@ export async function putObject(event: AWSCDKAsyncCustomResource.OnEventRequest)
   };
 }
 
-export async function deleteObject(event: AWSCDKAsyncCustomResource.OnEventRequest): Promise<AWSCDKAsyncCustomResource.OnEventResponse> {
+export async function deleteObject(event: AWSCDKAsyncCustomResource.OnEventRequest) {
```

Comment:
Why strip the explicit return type?

Reply:
because the actual return type is `Promise<AWSCDKAsyncCustomResource.OnEventResponse | void | undefined>` and it's a mouthful.